audio_analysis.go

```diff
@@ -0,0 +1,119 @@
+package spotify
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// AudioAnalysis contains a detailed audio analysis for a single track identified by its unique Spotify ID.
+// See https://developer.spotify.com/web-api/get-audio-analysis/
+//
+// Spotify's documentation is currently missing the object model for the AudioAnalysis.
```

Comment:
I would omit this.  Eventually they'll add documentation and we'll more than likely forget to come back and remove this.